src/core/ext/xds/xds_channel_stack_modifier.h

```diff
@@ -45,8 +45,7 @@ class XdsChannelStackModifier : public RefCounted<XdsChannelStackModifier> {
   explicit XdsChannelStackModifier(
       std::vector<const grpc_channel_filter*> filters)
       : filters_(std::move(filters)) {}
-  // Returns true on success, false otherwise.
-  bool ModifyChannelStack(ChannelStackBuilder* builder);
+  void ModifyChannelStack(ChannelStackBuilder& builder);
```

Comment:
Please update the comment on line 42 indicating how this is used.

Reply:
done